mqtt-bridge/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.strimzi.strimzi_mqtt_bridge</groupId>
+    <artifactId>strimzi_mqtt_bridge</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>19</maven.compiler.source>
+        <maven.compiler.target>19</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport</artifactId>
+            <version>4.1.87.Final</version>
```

Comment:
why using a different version from the other Netty dependencies?
Also it's important to use a property to define the Netty version in the `<properties>` section so that it can be reused easily across different Netty dependencies.

Reply:
Actually it was not intentional, I was using the IDE to automatically and the dependencies, I think it was not picking the same versions. 
but I have fixed it now.